test_runner/fixtures/neon_fixtures.py

```diff
@@ -1602,7 +1602,7 @@ def __init__(self, env: NeonEnv, port: PageserverPort, config_override: Optional
             # FIXME: there is a race condition between GC and detach, see
             # https://github.com/neondatabase/neon/issues/2442
             ".*could not remove ephemeral file.*No such file or directory.*",
-            # FIXME: These need investigation
```

Comment:
would the FIXME comment still apply to the others?

Reply:
Good point, only checked the two